docs/FAQ.md

```diff
@@ -0,0 +1,121 @@
+# Frequently Asked Questions
+
+## Basics
+
+### How do I choose between the `windows` and `windows-sys` crates?
+
+In general, `windows-sys` is meant to give users access to raw, zero-overhead bindings to the Windows API, while `windows` is meant as a more idiomatic, and when possible safer alternative. Below is a list of reasons you may choose `windows-sys` or `windows`.
+
+| Reason | `windows` | `windows-sys`|
+| --- | --- | --- |
+| Fast compile times possible are one of your top concerns| | ✅ |
+| You need COM or WinRT support. | ✅ | |
+| You would prefer to use APIs that feel idiomatic to Rust. | ✅ | |
+| Reason | You can't afford the very small cost of overhead | | ✅ |
```

Comment:
```suggestion
| You can't afford the very small cost of overhead | | ✅ |
```

Reply:
This one is a little vague. What overhead? The `windows` crate has compile-time overhead but that's already covered by "Fast compile times" above. It doesn't have any other overhead compared to `windows-sys` unless you enable the `alloc` feature so that's not a reason not to use it.

Reply:
That's probably true - the conversions that are done (e.g., converting out pointers + `HRESULT` to `windows::Result<T>` is transparent enough to the optimizer that it probably doesn't add anything. I can remove this.